### PR TITLE
Update README.md, fix broken link

### PR DESCRIPTION
**Pull request by broeder-j**
Replace broken dev_guide link with url.
Also add a sentence on what happens when cookiecutter is executed.

### CODE REVIEW

**Comment by apirogov** [comment]
It's not "broken", it's only broken if you browse the readme ;) It's working in the deployed website. 

To avoid copying the same files around, it takes the guide from the cookiecutter docs dir directly
(`docs_dir: "{{ cookiecutter.__project_slug }}/docs"`)

Have you tried your "fix"? I think if you'd do it like this, then you'll ACTUALLY break the link on the website (because the local md paths are mapped to urls, and if you never refer to a file, the site/URL will not be created in the first place).

I guess you just read the `README.md` on Github? the problem is, I can't "symlink" the file to the top level (as a better actual fix) because Github does not make symlinks transparent (it would link to a file that says this is a symlink to some path).

So, TL;DR: short of copying the file (and updating the copy automatically or manually whenever it changes) I see no solution.

I think maybe it's actually better to leave it as is until I set up the synchronized "demo repository" and link to that repository (containing also this file), instead to the guide.

**EDIT**:

Ah now I see you link directly to the GITHUB version of that page. but still, then it's not rendered properly, and I was thinking about using mkdoc syntax with boxes and stuff there. so also not a good idea.
